lib/modules/dosomething/dosomething_image/dosomething_image.module

```diff
@@ -139,7 +139,12 @@ function dosomething_image_get_themed_image_by_fid($fid, $style, $alt = '') {
   if (!$file) {
     return NULL;
   }
+
   $image = image_load($file->uri);
+  if (!$image) {
+    return NULL;
```

Comment:
`@return` statement should be fix in order to represent possible `NULL` value in return.